charts/bulk-scan-processor/values.yaml

```diff
@@ -27,7 +27,7 @@ java:
     BULK_SCANNING_DB_PORT: "5432"
     FLYWAY_URL: jdbc:postgresql://bulk-scan-processor-{{ .Values.global.environment }}.postgres.database.azure.com:5432/bulk_scan?sslmode=require
     FLYWAY_USER: bulk_scanner@bulk-scan-processor-{{ .Values.global.environment }}.postgres.database.azure.com
-    FLYWAY_NOOP_STRATEGY: "false"
+    FLYWAY_NOOP_STRATEGY: "true"
```

Comment:
why are we not running migrations?

Reply:
for preview it will be false no db migration for other envs it will be true i.e run migration provided db migration step is present.